lib/p2p/Pool.ts

```diff
@@ -579,8 +582,10 @@ class Pool extends EventEmitter {
     // request peer's orders
     peer.sendPacket(new packets.GetOrdersPacket({ pairIds: this.handshakeData.pairs }));
     if (this.config.discover) {
-      // request peer's known nodes only if p2p.discover option is true
-      peer.sendPacket(new packets.GetNodesPacket());
+      peer.getNodesTimer = setInterval(() => {
```

Comment:
It might be nice to be able to set the configured value to `0` to disable the scheduled GetNodes calls, if so here's where we'd want to make that check.

Reply:
good point